rules/windows/image_load/image_load_dll_rstrtmgr_suspicious_load.yml

```diff
@@ -0,0 +1,50 @@
+title: Load Of RstrtMgr.DLL By A Suspicious Process
+id: b48492dc-c5ef-4572-8dff-32bc241c15c8
+related:
+    - id: 3669afd2-9891-4534-a626-e5cf03810a61
+      type: derived
+status: experimental
+description: |
+    Detects the load of RstrtMgr DLL (Restart Manager) by a suspicious process.
+    This librairy has been used during ransomware campaigns to kill processes that would prevent file encryption by locking them (Conti Ransomware, Cactus Ransomware). It has also been seen recently in the BiBi wiper for Windows.
+    It could also be used for anti-analysis purposes by shut downing specific processes.
```

Comment:
```suggestion
    Detects the load of RstrtMgr DLL (Restart Manager) by a suspicious process.
    This library has been used during ransomware campaigns to kill processes that would prevent file encryption by locking them (e.g. Conti ransomware, Cactus ransomware). It has also recently been seen used by the BiBi wiper for Windows.
    It could also be used for anti-analysis purposes by shut downing specific processes.
```